test/clj/cider/nrepl/middleware/test_test.clj

```diff
@@ -261,7 +263,9 @@ The `988` value reflects that it times things correctly for a slow test.")
 (deftest print-object-test
   (testing "uses println for matcher-combinators results, otherwise invokes pprint"
     (is (= "{no quotes}\n"
-           (#'test/print-object (with-meta {"no" "quotes"} {:type :matcher-combinators.clj-test/mismatch})))
+           (#'test/print-object (matcher-combinators.clj-test/tagged-for-pretty-printing
+                                 '(not (match? 1 2))
+                                 {:matcher-combinators.result/value {"no" "quotes"}})))
```

Comment:
Updated this test to match this library's actual structure for a typical test failure.